### DISCUSSION TITLE
BTstack LEPeripheral - How to send data to other device

### DISCUSSION THREAD

**Discussion by DTSchneiderlein** [earlephilhower/arduino-pico#1525] [General]
I have tried BTstack LEPeripheral Example with a Pico w on one side and an Android device on the other. I can send individual bytes from the android device to the Pico (and read them out via the serial port), but I haven't figured out how to send anything from the Pico to the Android device. Is there any other way to do this yet?

**Comment by hacktronics** [comment]
I am also facing the same issue, from the raspberry pi sdk everything works, the temp sensor example. But using with Arduino and this library, the same functions are not working. I want to send response to the Android query. Did you find any solution to the same.
